backend/app/routers/authentication/authentication.py

```
@@ -0,0 +1,76 @@
+        ):
+            return {"message": "Verification code has expired. Try again"}
+
+        update_user_verification_status(email)
+
+        return {"message": "Email verified successfully"}
+    else:
+        return {"message": "Invalid verification code"}
+
+
+@router.get("/protected-endpoint")
+def protected_route(current_user: dict = Depends(get_current_user)):
```

Ruff (0.8.0)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)